plugins/trivy/cmd/trivy/main.go

```diff
@@ -27,9 +28,16 @@ const outputFile = image.TempDir + "/final-report.json"
  * Images or images recommendations that no longer belongs to that cluster are filtered out
  */
 func main() {
-	setLogLevel()
+	setLogLevel(os.Getenv("LOGRUS_LEVEL"))
 	setEnv()
-	lastReport, err := image.GetLastReport()
+
+	host := os.Getenv("FAIRWINDS_INSIGHTS_HOST")
```

Comment:
Minimized the usage of global variables, this allows for easier testing and makes functions pure